docs/rules/order.md

```diff
@@ -89,3 +91,48 @@ You can set the options like this:
 ```js
 "import/order": ["error", {"groups": ["index", "sibling", "parent", "internal", "external", "builtin"]}]
 ```
+
+### `newlines-between: [always|never]`:
+
+
+Enforces or forbids new lines between import groups:
+
+- If omitted, assertion messages will be neither enforced nor forbidden.
+- If set to `always`, a new line between each group will be enforced, and new lines inside a group will be forbidden.
+- If set to `never`, no new lines are allowed in the entire import section.
+
+For default group setting those will be valid:
+
+```js
+/* eslint import/order: ["error", {"newlines-between": "always"}] */
+var fs = require('fs');
+
+var index = require('./');
```

Comment:
Could you add
```js
var path = require('path');
```
otherwise it looks like you simply enforce a line between each import.

And since most examples in the repo us the import syntax, might be nice to use that instead?